sharded_queue/api.lua

```diff
@@ -77,8 +94,13 @@ function sharded_tube.take(self, timeout, options)
     if options == nil then
         options = {}
     end
+    
     options.tube_name = self.tube_name
 
+    if options and options.log_request == nil then
```

Comment:
`options` is always not nil

Reply:
Fix here https://github.com/tarantool/sharded-queue/pull/31/commits/1c83c6676da9cfbbe1abaa66b369bdfd26d12922